src/js/flags/glitch-fixes.ts

```diff
@@ -0,0 +1,56 @@
+import {FlagSection} from './flag';
+
+export const glitchFixFlags: FlagSection = {
+  section: 'Glitch Fixes',
+  text:
+      `Alternatively, glitches may be patched out of the game and made unusable.
+         These flags are exclusive with the flags that require the glitch.`,
+
+  flags: [
+    {
```

Comment:
It occurs to me that a common failure mode I've seen at least a few times is that I change the flag but don't update the check (i.e. the flag definition is changed to Ww but the lookup still checks Rw).  We could make this less likely by using a keyed object literal here:

```
export const glitchFixFlags = {
  ...,
  flags: {
    disableShopGlitch: {
      ...
    },
    ...
} as const;
```

Then in Flagset, we write

```
  disableShopGlitch() { return this.check(glitchFixFlags.flags.disableShopGlitch); }
```

It's a little verbose, and we could possibly remove some of the verbosity by pulling out the metadata:

```
export const glitchFixFlags = {
  [SECTION]: ...,
  [TEXT]: ...,
  disableShopGlitch: { ... }
}
```

or possibly an overarching `[METADATA]` symbol with an object underneath it.

Reply:
I don't know that I'll got down that road with this PR... One thought I had was refactoring the constructor of FlagSet to just store the flags themselves. Then, you could just use the objects as needed instead of doing lookups.

Reply:
Seems reasonable... we can look into it.  I'd like to get away from the string-based approach, and maybe clean up how we do hashing.  I suspect hashing the flag string is what was causing Dem and Syl their checksum mismatch problems the other day.

Reply:
Interesting...I'll have to dig in a bit more to look into that. Outside of that, is there anything I can look into to help understand some of the flow of the project? There's definitely a lot of manipulation of the bytes going on, and I have no idea what I'm looking at for most of it.

Reply:
I'm trying to name as many of the bytes as I can, FWIW :-)